### ISSUE TITLE
NewtonSoft.Json 12.0.1 support for NEST 2.5.8

### ISSUE THREAD

**Issue by sorcer1** [elastic/elasticsearch-net#3504]
Hello, 
It is possible to upgrade the NewtonSoft.Json dependency to 12.0.1 ?

There is (with the last version) a compile error :

```
Error	NU1107	
Version conflict detected for Newtonsoft.Json. Reference the package directly from the project to resolve this issue. 
 MyAssembly -> Newtonsoft.Json (>= 12.0.1) 
 MySearch -> NEST 2.5.8 -> Newtonsoft.Json (>= 11.0.1 && < 12.0.0).
```

**Comment by DaniilSokolyuk** [comment]
For NEST 5.6.4 too need upgrade

**Comment by Mpdreamz** [comment]
Hi @sorcer1 our 2.x version of the client is EOL, as is Elasticsearch 2.x https://www.elastic.co/support/eol.

We won't be doing anymore updates to this version.

@DaniilSokolyuk I've opened a PR to upgrade to 12 against 5.x today s this should be released next week.

**Comment by Mpdreamz** [comment]
Link to PR #3501

**Comment by Mpdreamz** [comment]
We just released [6.4.1 and 5.6.5](https://github.com/elastic/elasticsearch-net/releases) that bump the `Newtonsoft.Json` dependency to `12`. It might take a bit for them to show up on nuget.org.
